### PR TITLE
[develop] Base Project

### PR DESCRIPTION
**Pull request by JhordanSoto**
adding the base project

### CODE REVIEW

File: src/test/java/org/fundacionjala/trello/runner/Runner.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * Cucumber TestNG runner class.
+ */
+@CucumberOptions(
+        plugin = {"pretty"},
+        features = {"src/test/resources/features"},
+        glue = {"org.fundacionjala.trello"}
+)
+public final class Runner extends AbstractTestNGCucumberTests {
+    @Override
+    @DataProvider
```
Comment (MirkoFer-622) [inline src/test/java/org/fundacionjala/trello/runner/Runner.java]: Please consider to allow parallel execution of BDD Tests, adding: "@ DataProvider(parallel = true)"

**Review by MirkoFer-622 — CHANGES_REQUESTED** [review]

File: src/test/java/org/fundacionjala/trello/runner/Runner.java (hunk 1 above)
Comment (samuelloza) [reply src/test/java/org/fundacionjala/trello/runner/Runner.java]: Thanks, @MirkoFer-622 
I solved the observation

**Review by MirkoFer-622 — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+task executeBDDTests(type: Test) {
+    useTestNG()
+    options {
+        useDefaultListeners = false
+        jvmArgs = ['-Dcucumber.plugin=io.qameta.allure.cucumber6jvm.AllureCucumber6Jvm']
+        jvmArgs = ['-Dcucumber.plugin=rerun:build/target/rerun.txt']
+        jvmArgs = ['-Dcucumber.filter.tags=' + filterTags]
+        jvmArgs = ['-DcucumberThreadCount=' + cucumberThreadCount]
+        jvmArgs = ['-Dtoken=' + token]
+    }
```
Comment (marcos-x86) [inline build.gradle:95]: Please consider adding the remaining variables for authentication.

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+task reExecuteBDDTests(type: Test) {
+    useTestNG()
+    options {
+        useDefaultListeners = false
+        jvmArgs = ['-Dcucumber.plugin=io.qameta.allure.cucumber6jvm.AllureCucumber6Jvm']
+        jvmArgs = ['-Dcucumber.plugin=rerun:build/target/rerun.txt']
+        jvmArgs = ['-Dcucumber.features=@build/target/rerun.txt']
+        jvmArgs = ['-DcucumberThreadCount=' + cucumberThreadCount]
+        jvmArgs = ['-Dtoken=' + token]
+    }
```
Comment (marcos-x86) [inline build.gradle:112]: Please consider adding the remaining variables for authentication.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

**Review by maryangela — APPROVED** [review]

**Review by samuelloza — APPROVED** [review]

**Pull request merged**